xprof/xprof.rb.in

```diff
@@ -478,7 +478,7 @@ def setup_lttng(backends)
     channel_name = 'non-blocking-channel'
     exec("lttng enable-channel --userspace --session=#{lttng_session_uuid} #{channel_name}")
     exec("lttng add-context    --userspace --session=#{lttng_session_uuid} --channel=#{channel_name} -t vpid -t vtid")
-    exec("lttng enable-event  --userspace --session=#{lttng_session_uuid} --channel=#{channel_name} lttng_ust_sampling:*")
+    exec("lttng enable-event   --userspace --session=#{lttng_session_uuid} --channel=#{channel_name} lttng_ust_sampling:*")
```

Comment:
Is this change really needed, looks like just an extra space?

Reply:
yeah, just to align with the other :)

Reply:
Oh! makes sense.